### PR TITLE
Update to space rockets challenge

### PR DESCRIPTION
**Pull request by Wiingaard**

### CODE REVIEW

File: CHALLENGE.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+This last task is a chance for you to impress us. What you do is really up to
+you - show us your best skills, be creative and try not to spend more than a few
+hours. You can add a new feature (e.g. filtering of lists), or a whole new
+section to the app (explore the API, there is a ton we haven't used yet), add a UI test suite, improve the tooling or app's architecture, add
+beautiful animations, etc.
+
+#### Guidelines for Task 3
+
+- Treat this last task as a little hackathon - no need for tests and covering
```
Comment (lpozdniakova) [inline CHALLENGE.md:61]: For me there is a little discrepancy with what is written above. In "Impress us" part we say "add a UI test suite" and here "no need for tests". Should we rephrase it?

File: CHALLENGE.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  with a team, and the app is in production.
+- This app is not finished, there will be more pages added in the future, and
+  the feature you're implementing might be extended - consider that when
+  building your solution.
+- When in doubt about the design or functionality, make a decision yourself.
+- We estimate that this task can be done within a few hours - you can use the
+  PR description to list and discuss the things you decided to de-scope, potential
+  next steps etc.
+
+### 3. Impress us
```
Comment (lpozdniakova) [inline CHALLENGE.md:51]: Does this part mandatory or optional?

File: CHALLENGE.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+#### Guidelines for Task 2
+
+- Treat this as if you implemented a new feature in an app you work on together
+  with a team, and the app is in production.
+- This app is not finished, there will be more pages added in the future, and
+  the feature you're implementing might be extended - consider that when
+  building your solution.
+- When in doubt about the design or functionality, make a decision yourself.
+- We estimate that this task can be done within a few hours - you can use the
```
Comment (lpozdniakova) [inline CHALLENGE.md:47]: What if we ask candidates to let us know how much time did they spend?

File: CHALLENGE.md (hunk 1 above)
Comment (Wiingaard) [reply CHALLENGE.md:61]: This is just copied from the Frontend challenge, but we could change the wording in our challenge. 

The idea here is that in Task 1 and 2 we expect production ready quality code (including test ofc.), in Task 3 we expect hackathon quality code. If they choose to add UI tests in Task 3, then that'll off cause include tests 😊 

How about if change this sentence to:
> no need for covering all edge cases - although the code quality still matters. 

Do you think that makes more sense?

File: CHALLENGE.md (hunk 3 above)
Comment (Wiingaard) [reply CHALLENGE.md:47]: I actually think it's interesting to see how they would de-scope a project (like we sometimes have to do our selves in this job), plus it helps them to not be overambitious and accidentally spend too much. Maybe the wording is better if we write:

> Don't spend more than a few hours on this. You can use the PR description to list and discuss the things you decided to de-scope, potential next steps etc.

Is that better?

File: CHALLENGE.md (hunk 2 above)
Comment (Wiingaard) [reply CHALLENGE.md:51]: It's mandatory.

**Review by OsvaldoGabaldon — APPROVED** [review]
This is great man! 🙌🏼 feels so much better than the former challenge, more professional, more formal, and it's doable within a few hours. Let's just fine tune a few things and ship it! 🚀
